pnpm-lock.yaml

```diff
@@ -1762,6 +1762,9 @@ importers:
       config: 3.3.7
       jetpack-e2e-commons: link:../../../../../tools/e2e-commons
 
+  projects/plugins/super-cache:
+    specifiers: {}
+
```

Comment:
@Automattic/jetpack-crew Is this change expected? I'm not sure where it came from.

Reply:
It's harmless.

`projects/plugins/super-cache` was recently added, but since it has no JS dependencies yet it's optional and so the "lock file is up to date" check didn't flag it as missing. When you did something here that updated the lock file, it got added then.

And then when you did the trunk merge in c3295ff, that dropped the addition from your PR because #25687 did the addition along with its own changes.